zeroex/order_validator.go

```diff
@@ -225,80 +337,92 @@ func (o *OrderValidator) BatchValidate(rawSignedOrders []*SignedOrder) map[commo
 // - `Signature` contains a properly encoded 0x signature
 // - Validate that order isn't expired
 // Returns an orderHashToInfo mapping with all invalid orders added to it, and an array of the valid signedOrders
-func (o *OrderValidator) BatchOffchainValidation(signedOrders []*SignedOrder) (map[common.Hash]*OrderInfo, []*SignedOrder) {
-	orderHashToInfo := map[common.Hash]*OrderInfo{}
-	validSignedOrders := []*SignedOrder{}
+func (o *OrderValidator) BatchOffchainValidation(signedOrders []*SignedOrder) ([]*RejectedOrderInfo, []*SignedOrder) {
```

Comment:
optional: I think it would be more natural to re-arrange the return values so we return the valid orders first and the invalid ones second. This maps more nicely to the familiar `(result, error)` return values in Go.